projects/python-lz4/fuzz_lz4.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import atheris
+import lz4.frame
+
+def TestOneInput(data):
+    c =lz4.frame.compress(data)
+    d = lz4.frame.decompress(c)
+    assert(data == d)
+    return
+
+def main():
+    atheris.Setup(sys.argv, TestOneInput, enable_python_coverage=True)
+    atheris.Fuzz()
+
+if __name__ == "__main__":
```

Comment:
nit: we generally use single quotes in python files. I guess it doesn't matter to us so much since we don't maintain these.